env/player.py

```diff
@@ -15,10 +17,23 @@ class NoFlowerException(Exception):
     pass
 
 
-class Card:
-    FLOWER = "FLOWER"
-    SKULL = "SKULL"
-    hidden = "HIDDEN"
+class CannotLoadHiddenStateException(Exception):
+    pass
+
+
+@dataclass
+class PlayerState:
+    name: str
+    cards_hand: List[str]
+    cards_stack: List[str]
```

Comment:
🤔
Would be nice if those objects were List[Card]

Reply:
Used to be the case but mypy was not happy